src/server.rs

```diff
@@ -52,6 +53,17 @@ pub struct ServerPlugin {
     ///
     /// In practice updates will live at least `update_timeout`, and at most `2*update_timeout`.
     pub update_timeout: Duration,
+
+    /// If enabled, replication will be started automatically after connection.
+    ///
+    /// If disabled, replication should be started manually by sending the [`StartReplication`] event.
+    /// Until replication has started, the client and server can still exchange network events.
+    ///
+    /// All events from server will be buffered on client until replication starts, except the ones marked as independent.
+    /// See also [`ServerEventAppExt::make_independent`].
+    ///
+    /// [`ServerEventAppExt::make_independent`]: crate::core::event_registry::server_event::ServerEventAppExt::make_independent
```

Comment:
Maybe just inline it?
```suggestion
    /// See also [`ServerEventAppExt::make_independent`](crate::core::event_registry::server_event::ServerEventAppExt::make_independent).
```